source/mechanism.md

```diff
@@ -115,14 +115,46 @@ The primary aim with this deployment was to provide the [dxDAO](https://dutchx.r
 
 ### Some more details to the auctioneer powers:
 #### De- & whitelisting of tokens that generate Magnolia
-TODO
+Whitelisted tokens - if traded in a whitelisted pair - generate Magnolia, which can be used to reduce liquditiy contribution.
+Read above about [Magnolia](https://dutchx.readthedocs.io/en/latest/mechanism.html#magnolia), [Whitelist](https://dutchx.readthedocs.io/en/latest/mechanism.html#whitelist) and [Liquidity contribution](https://dutchx.readthedocs.io/en/latest/mechanism.html#liquidity-contribution).<br/>
+- Whitelisting (and de-whitelisting) can happen by token individually or by entire lists
+- It is not necessary that the token is either listed for trading or actively traded
+- The auctioneer calls the function *updateApprovalOfToken* in the [Token Whitelist contract](https://github.com/gnosis/dx-contracts/blob/master/contracts/base/TokenWhitelist.sol#L28)
+- There is no time lag to the execution of this function
+- It applies immediately and is effective upon users *claiming* their tokens of whitelisted tokens (if traded in a whitelisted pair), also if the auction had finished prior to triggering this function
+
 #### Changing the threshold to start auctions
-TODO
+Once this defined sellVolume (deposit) threshold is reached by both opposite auctions (each), and the prior auctions have finished, the new auctions will commence.
+- The auctioneer calls the function *updateThresholdNewAuction* of the [Master Contract](https://github.com/gnosis/dx-contracts/blob/master/contracts/DutchExchange.sol) (via the [Proxy](https://github.com/gnosis/dx-contracts/blob/master/contracts/DutchExchangeProxy.sol)
+- At the time of writing, this value is set at USD1000 (in uint USD)
```

Comment:
> (in uint USD)

I feel is unclear.
Maybe `In the contract 1 USD is represented as 1e18 uint256. E.g. USD1000 = 1000000000000000000000`

`uint` is an alias for the more explicit `uint256` in solidity, explicit is better imho

Reply:
```suggestion
- At the time of writing, this value is set at USD1000 *(Technical note: In the contract 1 USD is represented as 1e18 uint256. E.g. USD1000 = 1000000000000000000000)*
```